### DISCUSSION TITLE
Probleme mit Firmware 3.1

### DISCUSSION THREAD

**Discussion by Fetzy88** [fabian-lauer/dbus-shelly-3em-smartmeter#59] [Q&A]
Hallo Gemeinde,
nach Stunden des Bemühens muss ich euch um Hilfe bitten.

Ich habe einen Multiplus II GX mit der Firmware v3.11.
Dazu gesellt sich ein Shelly 3EM V1.14

Ich habe mich an die Anleitung gehalten, alles installiert.
Nur leider passiert rein garnichts.
Der Multiplus findet weder den Shelly, noch ändert sich etwas an der Visu.

Die Einstellungen der config file habe ich doppelt kontrolliert, dass passt.

Könnte das an der Firmware liegen?
Gibt es ein Fehlerprotokoll? Falls ja, wie kann ich darauf zugreifen?

Ich stamme aus der SPS Welt und tue mich im Moment noch etwas schwer.

Ich danke euch für eure Zeit und eure Mühen

Grüße

**Comment by berger-at-archtools-de** [comment]
Hallo Fetzy,
hast Du Dein Problem lösen können? Bei mir ist exakt das gleiche Problem. Multiplus 2 GX 48/3000, neues Venus OS large image installiert, das Skript von Fabian Lauer (https://github.com/fabian-lauer/dbus-shelly-3em-smartmeter) wurde ohne Fehlermeldung installiert, natürlich wurde die IP des Shelly 3EM korrekt in die config.ini eingetragen. Aber es wird kein Shelly in der Remote Console angezeigt.

Ich steh' jetzt ziemlich auf dem Schlauch ...

Gruß, Tom
